Gordon360/Services/MembershipRequestService.cs

```diff
@@ -113,150 +75,86 @@ public MEMBERSHIP ApproveRequest(int requestID)
         /// </summary>
         /// <param name="requestID">The membership request id</param>
         /// <returns>A copy of the deleted membership request</returns>
-        public REQUEST Delete(int requestID)
+        public async Task<RequestView> DeleteAsync(int requestID)
         {
-            var result = _context.REQUEST.Find(requestID);
-            if (result == null)
+            var request = await _context.REQUEST.FindAsync(requestID);
+            if (request == null)
             {
                 throw new ResourceNotFoundException() { ExceptionMessage = "The Request was not found." };
             }
-            _context.REQUEST.Remove(result);
-            _context.SaveChanges();
-            return result;
+
+            RequestView rv = _context.RequestView.FirstOrDefault(r => r.RequestID == request.REQUEST_ID);
```

Comment:
Could you use this service's `Get` method here?
```suggestion
            RequestView requestView = Get(request.REQUEST_ID);
```